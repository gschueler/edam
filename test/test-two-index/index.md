% Test two index

test two files with index