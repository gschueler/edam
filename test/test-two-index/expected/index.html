<!DOCTYPE html>
<html>
<head>
  <title>Test two index</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="generator" content="pandoc" />
  <link rel="stylesheet" href="style.css" type="text/css" />
</head>
<body>


<nav class="page top">
    <ul>
        <li class="current"><a href="index.html">Test two index</a></li>
        
        
        <li class="next"><a href="document.html">Chapter 1: Document</a></li>
    </ul>
</nav>
<div id="docbody">
<h1 class="title">Test two index</h1>
<p
>test two files with index</p
>
</div>

<nav class="page bottom">
    <ul>
        <li class="current"><a href="index.html">Test two index</a></li>
        
        
        <li class="next"><a href="document.html">Chapter 1: Document</a></li>
    </ul>
</nav>

</body>
</html>